thunder/tests/distributed/test_dtensor.py

```diff
@@ -29,6 +35,13 @@
 }
 
 
+# TODO: Use a better way to add opinfo rather than importing one at a time.
```

Comment:
Wouldn't this happen to be outdated, now that we use `get_opinfo`?

Reply:
Forgot to remove this, thanks!